lib/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -260,7 +260,7 @@ testRule(rule, {
     },
     {
       code: "@import 'test'; @include mixin(1) { @content; };",
-      fixed: "@import 'test';\n\n @include mixin(1) {\n\n @content; };",
```

Comment:
This changed is due to this [bit in the](https://github.com/stylelint/stylelint/blob/master/lib/utils/isStandardSyntaxAtRule.js#L11-L14) `isStandardSyntaxAtRule` util. I'd consider the existing behaviour a bug and the changes in this PR fix it, even though it's unrelated to Less Syntax.